packages/css-library/copy-uswds-color-tokens.js

```diff
@@ -0,0 +1,169 @@
+const fs = require('fs');
+
+// List of color tokens to "pass through"
+const tokensToKeep = {
+  'blue': {
+    file: 'blue.json',
+    colors: [
+      'blue-5',
+      'blue-10',
+      'blue-20',
+      'blue-30',
+      'blue-40',
+      'blue-50',
+      'blue-60',
+      'blue-70',
+      'blue-80',
+      'blue-90',
+      'blue-vivid-50',
+      'blue-vivid-60',
+    ],
+  },
+  'blue-cool': {
+    file: 'blue-cool.json',
+    colors: [
+      'blue-cool-vivid-5',
+      'blue-cool-vivid-20',
+      'blue-cool-vivid-40',
+      'blue-cool-vivid-60',
+    ],
+  },
+  'blue-warm': {
+    file: 'blue-warm.json',
+    colors: ['blue-warm-vivid-60', 'blue-warm-vivid-70', 'blue-warm-vivid-80'],
+  },
+  'cyan': {
+    file: 'cyan.json',
+    colors: ['cyan-vivid-30'],
+  },
+  'gold': {
+    file: 'gold.json',
+    colors: ['gold-vivid-5'],
+  },
+  'gray': {
+    file: 'gray.json',
+    colors: [
+      'gray-1',
+      'gray-2',
+      'gray-3',
+      'gray-4',
+      'gray-5',
+      'gray-10',
+      'gray-20',
+      'gray-30',
+      'gray-40',
+      'gray-50',
+      'gray-60',
+      'gray-70',
+      'gray-80',
+      'gray-90',
+      'gray-100',
+    ],
+  },
+  'gray-cool': {
+    file: 'gray-cool.json',
+    colors: [
+      'gray-cool-1',
+      'gray-cool-2',
+      'gray-cool-3',
+      'gray-cool-4',
+      'gray-cool-5',
+      'gray-cool-10',
+      'gray-cool-20',
+      'gray-cool-30',
+      'gray-cool-40',
+      'gray-cool-50',
+      'gray-cool-60',
+      'gray-cool-70',
+      'gray-cool-80',
+      'gray-cool-90',
+    ],
+  },
+  'orange': {
+    file: 'orange.json',
+    colors: ['orange-40'],
+  },
+  'violet': {
+    file: 'violet.json',
+    colors: ['violet-vivid-70'],
+  },
+  'yellow': {
+    file: 'yellow.json',
+    colors: ['yellow-5'],
+  },
+};
+// Desired prefix/namespace for colors
+const prefix = 'uswds-system-color';
```

Comment:
Is this a specific prefix that we need to use? Or just a general way to identify those tokens as "uswds"? Just a nit but could this just be shortened to `uswds`?  It just seems like some of the tokens get really long with it: 

`--uswds-system-color-blue-warm-vivid-80: #162e51;`

Reply:
This is the prefix that @humancompanion-usds indicated he wanted to use, but easy to change it if we want to.